pathfinder/paizo/player_companion/blood_of_the_elements/bote_abilities_class.lst

```diff
@@ -0,0 +1,37 @@
+# CVS $Revision: $ $Author: $ -- Fri Jan 29 20:50:52 2016 -- reformated by PCGen PrettyLST v6.06.00
+SOURCELONG:Blood of the Elements	SOURCESHORT:BOTE	SOURCEWEB:http://paizo.com/products/btpy978p?Pathfinder-Player-Companion-Blood-of-the-Elements	SOURCEDATE:2014-06
+# ORIGINAL ENTRY: Gwen Thebich (evilpixie87)
+# ORIGINAL ENTRY DATE: 4 April 2016
+
+
+###Verify Crossblooded works with bloodlines
+
+
+# ==============================
+# Mutated Bloodlines
+#===============================
+
+
+Lifewater (Wildblooded Elemental Water) Bloodline				CATEGORY:Special Ability	TYPE:SorcererClassFeatures.SpecialQuality.ClassSpecialization.SorcererBloodline		DESC:Your soul is infused with the restorative and life-giving powers of elemental water.							ABILITY:Special Ability|AUTOMATIC|Bloodline Arcana (Lifewater)|Lifewater Bloodline Bonus Spells				ABILITY:Special Ability|AUTOMATIC|Lifewater Bloodline ~ Elemental Ray|PREVAREQ:Sorcerer_CF_BloodlinePower1,0			ABILITY:Special Ability|AUTOMATIC|Lifewater Bloodline ~ Elemental Resistance|PREVAREQ:Sorcerer_CF_BloodlinePower3,0|PREVARGTEQ:BloodlineLVL,3|PREVARLT:BloodlineLVL,20					ABILITY:Special Ability|AUTOMATIC|KEY:Lifewater Bloodline ~ Lifewater Stream|PREVAREQ:Sorcerer_CF_BloodlinePower9,0|PREVARGTEQ:BloodlineLVL,9				ABILITY:Special Ability|AUTOMATIC|Lifewater Bloodline ~ Elemental Movement|PREVAREQ:Sorcerer_CF_BloodlinePower15,0|PREVARGTEQ:BloodlineLVL,15			ABILITY:Special Ability|Lifewater Bloodline ~ Elemental Body|PREVAREQ:Sorcerer_CF_BloodlinePower20,0|PREVARGTEQ:BloodlineLVL,20								BONUS:ABILITYPOOL|Lifewater Bloodline Feat|BloodlineFeatCount			CSKILL:Knowledge (Planes)		SOURCEPAGE:p.15
```

Comment:
Missing "KEY:Sorcerer Bloodline ~ Lifewater" same for the following. Important to use KEY to identify class, since later sources have a habit of re-using bloodlines for other classes.